index.js

```diff
@@ -111,7 +113,11 @@ module.exports = function makeFetch (opts = {}) {
       await archive.ready()
 
       if (version) {
-        archive = archive.checkout(version)
+        if (NUMBER_REGEX.test(version)) {
+          archive = await archive.checkout(version)
```

Comment:
Should this maybe be `.checkout(parseInt(version, 10))`? Or does it work with string versions?

Reply:
Works fine, tested. :-)